python-threatexchange/threatexchange/fetcher/fetch_api.py

```diff
@@ -109,33 +146,56 @@ def report_opinion(
 
         Most implementations will want a full replacement specialization, but this
         allows a common interface for all uploads for the simplest usecases.
+
+        This is an optional API, and places that use it should catch
+        the NotImplementError.
         """
         raise NotImplementedError
 
     def report_true_positive(
         self,
+        collab: CollaborationConfigBase,
         s_type: t.Type[SignalType],
         signal: str,
-        metadata: state.FetchedSignalDataBase,
+        metadata: state.FetchedSignalMetadata,
     ) -> None:
         """
         Report that a previously seen signal was a true positive.
 
         This is an optional API, and places that use it should catch
         the NotImplementError.
         """
-        raise NotImplementedError
+        self.report_opinion(
+            collab,
+            s_type,
+            signal,
+            state.SignalOpinion(
+                owner=self.get_own_owner_id(collab),
+                category=state.SignalOpinionCategory.TRUE_POSITIVE,
+                tags=set(),
+            ),
+        )
 
     def report_false_positive(
         self,
+        collab: CollaborationConfigBase,
         s_type: t.Type[SignalType],
         signal: str,
-        metadata: state.FetchedSignalDataBase,
+        _metadata: state.FetchedSignalMetadata,
```

Comment:
Probably not use _ for an interface method?

Reply:
It generates a lint error because it's unused otherwise :/ Maybe that has gone away now that I have children...